core/idl/wallet/bitcoin/bitcoin_like_wallet.djinni

```diff
@@ -112,6 +112,12 @@ BitcoinLikeBlock = interface +c {
     getTime(): date;
 }
 
+BitcoinLikeSignature = record {
```

Comment:
Even though those fields make sense for DER / ECDSA, it’s a bit cryptic here when reading the IDL documentation. Can you provide some documentation for this type and its fields?

Reply:
Thank you!